index.html

```diff
@@ -3794,6 +3801,71 @@ <h3><dfn>Get Window Handles</dfn></h3>
 </aside>
 </section> <!-- /Get Window Handles -->
 
+<section>
+<h3><dfn>Create Window</dfn></h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/window/new</td>
+ </tr>
+</table>
+
+<p>Create a new <a>top level browsing context</a>.
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+ <li><p>If the implementation does not support creating new top-level
+  browsing contexts, return <a>error</a> with <a>error code</a>
+  <a>unsupported operation</a>.
+
+ <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+  return <a>error</a> with <a>error code</a> <a>no such window</a>.
+
+ <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
+
+ <li><p>Let <var>type hint</var> be the result of <a>getting the property</a>
+  "<code>type</code>" from the <var>parameters</var> argument.
+
+ <li><p>Create a new <a>top-level browsing context</a> by running
+  the <a>window open steps</a> with <var>url</var> set to
+  "<code>about:blank</code>", <var>target</var> set to the empty string,
+  and <var>features</var> set to "<code>noopener</code>" and the user
+  agent configured to create a new browing context. If <var>type
+  hint</var> has the value "<code>tab</code>" and the implementation
+  supports multiple browsing context in the same OS window, the new
+  browsing context should share an OS window with the <a>current
```

Comment:
What I miss here is what should happen when the implementation supports tabs, but not windows (eg. Fennec), and the typ is set to `window`. Should it then default to `tab`? I would say so. The implementation should select the best possible option.

Reply:
That is what the spec text says.

Reply:
No, the text above only covers the case when `tab` is given as type hint, but not `window`. And specifically when the implementation cannot open a window, but could perfectly share an OS window.

Btw. is `type hint` a valid variable we can use here? Or should it just be `type`, whereby that is a reserved keyword in most languages.

Reply:
I fixed the text.